### DISCUSSION TITLE
Performance metrics

### DISCUSSION THREAD

**Discussion by caomre** [imartinez/privateGPT#1524] [Q&A]
Hello.

Does anyone have any performance metrics for PrivateGPT? E.g. what is good, what is not good.
For example, I've managed to set it up and launch on AWS/Linux (p2.8xlarge instance, 32 vCPUs, 488 Gb RAM, 8 GPUs). Then I fed a 10 Mb text file to it, and it took almost 30 min to digest it... Is it about right or I messed up something during installation?

Thanks.

**Comment by lolo9538** [comment]
it's slow, I have installed it on my own pc with a 4070ti and it only takes a few minutes for such a document to ingest. (32GB of RAM, processor with 6 cores 3.7 Ghz).
Make sure you are using the power of your GPU when running it, there were specific steps for windows, I don't know about Linux.
